Hogajama/hogajama-angular-frontend/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1 @@
+mp.openapi.extensions.enabled=false
```

Comment:
Habe OpenAPI Subsystem hinzugefügt (`03_set_microprofile_openapi.cli`).

Reply:
Alles klar! Ich hoffe es war in Ordnung, dass ich dir bei der Server config nicht hineingepfuscht habe.